dashboard/v1.1/src/pages/Ping/index.tsx

```diff
@@ -1,7 +1,151 @@
-import React, { FC } from 'react';
+import React, { FC, useState, useEffect } from 'react';
+import { useFetch } from '../../utils/useFetch';
+import { service_states, HOST_IP } from '../../utils/types';
+import { Spinner } from 'reactstrap';
+import { Card, CardContent, TextField } from '@material-ui/core';
+import {
+  RoutesSummary,
+  QueryResponse,
+  chartData,
+  APIQueryResponse
+} from '../../utils/queryTypes';
+import Autocomplete from '@material-ui/lab/Autocomplete';
+import Alert from '@material-ui/lab/Alert';
+import { formatTime } from '../../utils/brt';
+import Ping from '../Monitoring/Ping';
 
-const Ping: FC<{}> = () => {
-  return <div>Ping Page</div>;
+interface showChartsDataParam {
+  pingMin: chartData[];
+  pingMean: chartData[];
+  pingMax: chartData[];
+}
+
+const PingModule: FC<{}> = () => {
+  const [routesDetails, setRoutesDetails] = useState<RoutesSummary>();
+  const [value] = useState(routesDetails?.testServicesRoutes[0]);
+  const [inputValue, setInputValue] = useState('');
+  const [showCharts, setShowCharts] = useState(false);
+  const [pingData, setPingData] = useState<showChartsDataParam>();
+
+  const { response, error } = useFetch<service_states>(
+    `${HOST_IP}/service-state`
+  );
+
+  useEffect(() => {
+    fetch(`${HOST_IP}/routes-summary`)
+      .then(res => res.json())
+      .then((response: { status: string; data: RoutesSummary }) => {
+        console.log(response.data);
+        setRoutesDetails(response.data);
+      });
+  }, []);
+
+  async function getChartsData(v: any) {
+    var res = '';
+    if (v.startsWith('https://www.')) {
+      res = v.replace('https://www.', '');
+    } else if (v.startsWith('https://')) {
+      res = v.replace('https://', '');
+    }
+
```

Comment:
cases for `http` should also be there. Its okay for now, but later on, this needs to be there.